package.json

```diff
@@ -53,6 +53,7 @@
     "connect-flash": "^0.1.1",
     "convict": "^4.0.0",
     "cookie-session": "^2.0.0-alpha.2",
+    "json-2-csv": "^3.2.0",
```

Comment:
This should be part of another PR, right?

Reply:
d'oh